### PR TITLE
Ux css community

### PR DESCRIPTION
**Pull request by yulldesign**

### CODE REVIEW

File: src/Pim/Bundle/UIBundle/Resources/public/css/buttons.less [hunk 1]
```diff
@@ -380,7 +391,7 @@
 .navbar-title .sub-title .pimmultiselect {
   height: 34px;
   line-height: 34px;
-  margin-top: -2px !important;
+  margin-top: -0px !important;
```
Comment (skeleton) [inline src/Pim/Bundle/UIBundle/Resources/public/css/buttons.less]: remove `-` and `px`

File: src/Pim/Bundle/UIBundle/Resources/public/css/form.less [hunk 2]
```diff
@@ -882,9 +882,16 @@ div.dropdown-menu.channel {
   cursor: default;
   opacity: 0;
   .label-warning {
-    color: @textColor + #333;
+    color: #777;
```
Comment (skeleton) [inline src/Pim/Bundle/UIBundle/Resources/public/css/form.less]: why not @color-form-info ?

**Comment by phaseinducer** [comment]
GTM

**Comment by sumbobyboys** [comment]
GTM

**Pull request merged**